testutils/infra/cli.py

```diff
@@ -12,12 +12,16 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
+from collections import namedtuple
+
 from testutils.infra.container_manager.docker_manager import DockerNamespace
 from testutils.infra.container_manager.kubernetes_manager import (
     KubernetesNamespace,
     isK8S,
 )
 
+Service = namedtuple("Service", "bin_path data_path")
```

Comment:
Nitpick: I would name this `Microservice` or `Serverservice`, to clearly differentiate from "systemd service" concept that we have in this test suite